src/app/components/modals/modal-language/modal-language.component.html

```diff
@@ -9,6 +9,7 @@ <h4 class="modal-title underline-accent stickyHeader">{{modal.modal_language_tit
         </mat-option>
       </mat-select>
     </div>
+    <mat-checkbox color="warn" (click)="saveDefault()">Set this as my default language</mat-checkbox>
```

Comment:
No text directly in code, put it in the translation files (english, french and arabic)